claripy/strings.py

```diff
@@ -100,17 +108,21 @@ def StrSuffixOf(suffix, input_string):
     """
     return re.match(r'.*' + suffix.value + '$', input_string.value) is not None
 
+
 def StrIndexOf(input_string, substring, startIndex, bitlength):
     """
-    Return True if the concrete value of the input_string ends with suffix
-    otherwise false.
+    Return a concrete BitVector of bitlength size. Its value is the index of
+    the first occurrence of substring searching from startIndex if found
+    otherwise is -1.
 
     :param input_string: the string we want to check
     :param substring: the substring we want to find the index
     :param startIndex: the index to start searching at
-    :param bitlength: bitlength of the bitvector representing the index of the substring
+    :param bitlength: bitlength of the bitvector representing the index of the
+                      substring
 
-    :return BVV: index of the substring in bit-vector representation or -1 in bitvector representation
+    :return BVV: index of the substring in bitvector representation or -1 in
+                 bitvector representation
```

Comment:
BVV is not a type; you want to say BV here. Also "bitvector representation" is a bit purple-prose for API documentation. Just say it's a bitvector.